docs/index.md

```diff
@@ -28,6 +28,194 @@ Core utilities such as Tracer, Logger, Metrics, and Event Handler will be availa
 
 ## Installation
 
+Powertools is available in the following formats:
+
+* **Lambda Layer**: [**arn:aws:lambda:{region}:094274105915:layer:AWSLambdaPowertoolsNodeJS:1**](#){: .copyMe}:clipboard:
+* **NPM**: **`npm install @aws-lambda-powertools/tracer @aws-lambda-powertools/metrics @aws-lambda-powertools/logger`**
+
+???+ hint "Support this project by using Lambda Layers :heart:"
+    Lambda Layers allow us to understand who uses this library in a non-intrusive way. This helps us justify and gain future investments for other Lambda Powertools languages.
+
+    When using Layers, you can add Lambda Powertools as a dev dependency (or as part of your virtual env) to not impact the development process.
```

Comment:
the `virtual env` wording is Python specific, let's either remove the text within the parentheses or specify the `devDependencies` or `npm i -D ....`